internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/CoordinatorGrpcRetryableClient.java

```diff
@@ -85,7 +85,7 @@ public void sendAppHeartBeat(RssAppHeartBeatRequest request, long timeoutMs) {
         "send heartbeat to coordinator");
   }
 
-  public void registerApplicationInfo(RssApplicationInfoRequest request, long timeoutMs) {
+  public void registerApplicationInfo(RssApplicationInfoRequest request) {
```

Comment:
As nearly all of the API of this class are similar with `CoordinatorClient`, how about forward further to let this class `implements CoordinatorClient`, then `CoordinatorClientFactory#createCoordinatorClient` can create one of the  concrete implementation within `CoordinatorGrpcRetryableClient` or others.

Reply:
done